tracer/test/Datadog.Trace.Tests/Telemetry/config_norm_rules.json

```diff
@@ -671,5 +671,6 @@
   "DD_TRACE_BAGGAGE_MAX_BYTES": "trace_baggage_max_bytes",
   "DD_TRACE_BYPASS_HTTP_REQUEST_URL_CACHING_ENABLED": "trace_bypass_http_request_url_caching_enabled",
   "DD_TRACE_INFERRED_PROXY_SERVICES_ENABLED": "inferred_proxy_services_enabled",
+  "DD_TRACE_DATA_PIPELINE_ENABLED": "trace_data_pipeline_enabled",
```

Comment:
We need to make sure to do a PR to dd-go to add this to the intake. Are we happy with the name? Data Pipeline is definitely what we're calling the feature across languages?